backend/src/akvo/lumen/component/keycloak.clj

```diff
@@ -6,6 +6,8 @@
             [cheshire.core :as json]
             [clj-http.client :as client]
             [integrant.core :as ig]
+            [clojure.spec.alpha :as s]
+            [akvo.lumen.specs.components :refer (integrant-key)]
```

Comment:
Since we used vectors most of the places, is there a reason for using the list notation?